### PR TITLE
fix(tests) : Remove old JS based functional tests

### PR DESCRIPTION
**Pull request by Raunak1203**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
This PR removes old JS based functional tests
- [x] What issue/task does this PR references?
https://github.com/fabric8-ui/fabric8-planner/issues/2599
- [ ] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @Raunak1203 for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-ui/fabric8-planner/blob/3c07f0b1de7b45f142defd319c812f83829f520d/.ike-prow/test-keeper.yaml).

File: runtime/src/tests/work-item/work-item-list/workitem-naughty-string.spec.js [hunk 1]
```diff
@@ -1,102 +0,0 @@
...
-
-describe('Work item list', function () {
-  var page, items, browserMode;
-  var until = protractor.ExpectedConditions;
-  var waitTime = 30000;
-
-/* Troublesome strings reference: https://github.com/minimaxir/big-list-of-naughty-strings */
-var UnicodeSymbols = "Ω≈ç√∫˜µ≤≥÷åß∂ƒ©˙∆˚¬…æœ∑´®†¥¨ˆøπ“‘¡™£¢∞§¶•ªº–≠¸˛Ç◊ı˜Â¯˘¿ÅÍÎÏ˝ÓÔÒÚÆ☃Œ„´‰ˇÁ¨ˆØ∏”’`⁄€‹›ﬁﬂ‡°·‚—±⅛⅜⅝⅞ЁЂЃЄЅІЇЈЉЊЋЌЍЎЏАБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯабвгдежзийклмнопрстуфхцчшщъыьэюя٠١٢٣٤٥٦٧٨٩";
-var TwoByteCharacters = "田中さんにあげて下さいパーティーへ行かないか和製漢語部落格사회과학원 어학연구소찦차를 타고 온 펲시맨과 쑛다리 똠방각하社會科學院語學研";
-var JapaneseEmoticons = "ヽ༼ຈل͜ຈ༽ﾉ ヽ༼ຈل͜ຈ༽ﾉ(｡◕ ∀ ◕｡)｀ｨ(´∀｀∩__ﾛ(,_,*)・(￣∀￣)・:*:ﾟ･✿ヾ╲(｡◕‿◕｡)╱✿･ﾟ,。・:*:・゜’( ☻ ω ☻ )。・:*:・゜’(╯°□°）╯︵ ┻━┻)(ﾉಥ益ಥ）ﾉ﻿ ┻━┻┬─┬ノ( º _ ºノ)( ͡° ͜ʖ ͡°)";
```
Comment (rgarg1) [inline runtime/src/tests/work-item/work-item-list/workitem-naughty-string.spec.js]: @Raunak1203 

Have you ported the naughty strings tests to the new TS tests?

Comment (Raunak1203) [reply runtime/src/tests/work-item/work-item-list/workitem-naughty-string.spec.js]: @rgarg1 here is the PR https://github.com/fabric8-ui/fabric8-planner/pull/2733

Comment (rgarg1) [reply runtime/src/tests/work-item/work-item-list/workitem-naughty-string.spec.js]: Excellent

**Pull request merged**